changelogs/fragments/2731-mh-cmd-locale.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - module_helper module utils - ``CmdMixin`` must also use ``LC_ALL`` to enforce locale choice (https://github.com/ansible-collections/community.general/issues/2715).
```

Comment:
```suggestion
  - module_helper module utils - ``CmdMixin`` must also use ``LC_ALL`` to enforce locale choice (https://github.com/ansible-collections/community.general/pull/2731).
  - xfconf - also use ``LC_ALL`` to enforce locale choice (https://github.com/ansible-collections/community.general/issues/2715).
```

Reply:
In that case should we include `cpanm` and `snap` as well? they were also affected by this change

Reply:
Sounds good.